### ISSUE TITLE
Kode evalueringsfejl i  01-intor-to-r

### ISSUE THREAD

**Issue by enerammer** [KUBDatalab/R-intro#46]
Der står 
function (x, digits = 0) 
NULL

**Comment by chrbknudsen** [comment]
Nope. Det er ikke en fejl. Det er outputtet af args(), jf dokumentationen:

Value
For a closure, a closure with identical formal argument list but an empty (NULL) body.

For a primitive (function), a closure with the documented usage and NULL body. Note that some primitives do not make use of named arguments and match by position rather than name.

NULL in case of a non-function.

Og det bliver jeg ikke supermeget klogere af hvis jeg skal være ærlig...
Men jeg tror (understreger tror) at vi skal forstå det som, at når vi har defineret en funktion:
min_funktion <- function(x, parameter = 47){
   x*parameter
}

Så er resultatet af args(min_funktion) :
function(x, parameter = 47)

Og NULL står så i stedet for:
{
   x*parameter
}

**Comment by chrbknudsen** [comment]
Bør der være en omtale af NULL?

**Comment by enerammer** [comment]
Det bør der helt klart være - synes jeg

**Comment by chrbknudsen** [comment]
den er assignet til dig - ser jeg efter jeg har skrevet en omtale...

**Comment by enerammer** [comment]
Du tager den bare
